homeassistant/components/watergate/quality_scale.yaml

```diff
@@ -0,0 +1,43 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
```

Comment:
Looking at the repository https://github.com/watergate-ai/watergate-local-api-python, we don't build and publish the dependency in the CI. That is required to increase transparency and security. For example, the integration uses 2024.2.1, while the repository only has 2024.1.11.

Reply:
Done!